package.json

```diff
@@ -14,6 +14,9 @@
   "author": "Saribeg",
   "license": "ISC",
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/material": "^5.10.10",
```

Comment:
Please, delete these dependencies as there are no neediness for them in server package.json